docs/DeveloperGuide.md

```diff
@@ -147,7 +147,89 @@ Classes used by multiple components are in the `seedu.storemando.commons` packag
 
 ## **Implementation**
 
-This section describes some noteworthy details on how certain features are implemented.
+### Help feature
+The help feature redirects the user to StoreMando's User Guide. If the user is connected to a internet access, StoreMando
+will redirect it's User Guide through opening another browser. Otherwise, it will have a pop out window with the User 
+Guide link. This will save the user the hassle of locating the documentation. <br>
+The help command has the following format :`help`.
+
+:information_source: : Things to note
+- Even though the help command expects the user input to contains the `help` command keyword, it still allows users to
+append arguments. However, the arguments will not be parsed by StoreMando.
+
+#### Implementation
+![HelpSequenceDiagram](images/HelpSequenceDiagram.png)
+Step 1: The user inputs `help` wanting to be redirected to the User Guide to learn more about StoreMando's command. <br>
+Step 2: The input is passed to `LogicManager` in a form of a String. <br>
+Step 3: The input string is passed to `StoreMandoParser` for it to be parsed. <br>
+Step 4: The input string is separated into command keyword and arguments containing the prefixes with the updated item's
+attribute. <br>
+Step 5: StoreMandoParser recognise that it is a `Help` command and creates a `HelpCommand` object. <br>
+Step 6: The `HelpCommand` is pass back to `LogicManager`. <br>
+Step 7: `LogicManager` then execute the `HelpCommand`. <br>
+Step 8: `HelpCommand` will create a `CommandResult` and pass to `LogicManager` with message stating 
+"Opened user guide information." <br>
```

Comment:
do not put <br> in your code, that is against the coding standards for md files :)

Reply:
All right noted!